lmdeploy/pytorch/engine/engine.py

```diff
@@ -488,10 +488,10 @@ def async_sampling_logits(self, logits: torch.Tensor,
 
         def __get_last_logits():
             """get last logits."""
-            if inputs.is_decoding:
```

Comment:
why change this if statement?

Reply:
if `return_logits` is fails, we will out include logits except the last token.

Reply:
`return_logits=False` will not include non-output logits.